### PR TITLE
Authentication docs - phase 1

### PR DESCRIPTION
**Pull request by psiinon**
Index and the first 2 pages - lots more to do, but I think its work publishing them as they get written.

This is intended to be the definitive guide to ZAP auth, so if you spot anything missing, wrong or that could be explained better then just shout ;)
Screenshot 2021-11-09 at 12-36-27 OWASP ZAP – ZAP Authentication

Screenshot 2021-11-09 at 12-36-36 OWASP ZAP – Authentication - Make your Life Easier

Screenshot 2021-11-09 at 12-36-46 OWASP ZAP – Authentication - Manual

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

**Review by kingthorin — COMMENTED** [review]
I only see one minor typo, otherwise looks good to me.

File: site/content/docs/authentication/make-your-life-easier.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+If your app can be run with full functionality and without authentication then just do that - in this case you are testing the app, not the authentication controls.
+
+Single Sign On systems can be especially hard to work with. 
+If you can use a simpler authentication mechanism like HTTP auth or a simple POST form then do that - these options will be much
+easier to set up and much less likely to break your testing.
+
+Two factor authentication (2AF) can be even harder to work with.
+ZAP does not work by magic - if you want to perform automated scanning but need a 2FA token then you are going to need to be able to 
+get that token to ZAP. If you cannot to that then you will not be able to automate your authentication.
```
[the remark this answers is not in the document]
Comment (psiinon) [reply site/content/docs/authentication/make-your-life-easier.md]: Done, and made the case of the links on the top page consistent...

**Review by kingthorin — APPROVED** [review]
Thanks

File: site/content/docs/authentication/make-your-life-easier.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kingthorin) [reply site/content/docs/authentication/make-your-life-easier.md]: Excellent

**Pull request merged**